tools/model_tools/src/omz_tools/omz_converter.py

```diff
@@ -20,7 +20,7 @@
 import string
 import sys
 
-from openvino.runtime import Core, serialize
+from openvino import Core, serialize
```

Comment:
I think it is also will be good to use save_model instead of serialize as it is deprecated